### PR TITLE
v08

### PR DESCRIPTION
**Pull request by mjethanandani**
Updates based on reviewing comments from Martin and Jan.

### CODE REVIEW

File: draft/draft-ietf-netconf-https-notif.xml [hunk 1]
```diff
@@ -193,7 +192,7 @@
      the receiver.";
   leaf-list receiver-capability {
     type string {
-      pattern "urn:ietf:capability:https-notif-receiver:*";
+      pattern "inet:uri";
```
Comment (kwatsen) [inline draft/draft-ietf-netconf-https-notif.xml]: This should be "type inet.uri" (not a "pattern: statement)

File: draft/draft-ietf-netconf-https-notif.xml [hunk 2]
```diff
@@ -204,12 +203,14 @@
             </artwork>
           </figure>
         </t>
+	<t>The publisher MUST ignore any capabilities that it does not recognize.
+	</t>
```
Comment (kwatsen) [inline draft/draft-ietf-netconf-https-notif.xml:206]: May be to prepend "As it is possible the receiver may return custom capability URIs, ..."

PS: unsure if "custom" is the best word.

File: draft/draft-ietf-netconf-https-notif.xml [hunk 3]
```diff
@@ -373,7 +374,7 @@ INSERT_TEXT_FROM_FILE(../bin/ietf-subscribed-notif-receivers@YYYY-MM-DD.yang,69)
           communicate with the receiver. The module augments the
           "ietf-subscribed-notif-receivers" module to define a transport specific
           receiver. </t>
-        <t>As mentioned earlier, it uses a POST method to deliver the
+        <t>As mentioned earlier, it uses an POST method to deliver the
```
Comment (kwatsen) [inline draft/draft-ietf-netconf-https-notif.xml]: I don't think this article is correct.

File: src/yang/ietf-https-notif-transport.yang [hunk 4]
```diff
@@ -106,7 +106,7 @@ module ietf-https-notif-transport {
         }
         description
           "Augmentation to add a receiver-specific path for the
-           'capabilities' and 'relay-notification' resources.";
+          'capabilities' and 'relay-notification' resources.";
```
Comment (kwatsen) [inline src/yang/ietf-https-notif-transport.yang]: I don't think this outdent is correct.

**Review by kwatsen — CHANGES_REQUESTED** [review]

**Comment by kwatsen** [comment]
I was expecting more changes.  What about the Title?

**Comment by kwatsen** [comment]
good enough but, FYI, there's no need to set the "abbrev" attribute if it's the same value as the title...

**Comment by mjethanandani** [comment]
> good enough but, FYI, there's no need to set the "abbrev" attribute if it's the same value as the title...

Looks like the "abbrev" text is too long. How about "HTTPS YANG Notifications"?

**Comment by kwatsen** [comment]
Or maybe "HTTPS Notif Transport" ???

Either way is fine by me.

**Pull request merged**